src/CSSClassRepr.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @flow
+ */
+
+export class CSSClassRepr {
+  className: string;
+  repr: any;
```

Comment:
If this is "representation", can we call it fullname? And CSSClassRepresentation? Maybe it is long to type, but more obvious